activerecord/test/cases/calculations_test.rb

```diff
@@ -234,6 +234,7 @@ def test_apply_distinct_in_count
     end
 
     queries.each do |query|
+      next if query == "SHOW max_identifier_length"
```

Comment:
I think this needs a comment to explain why it's there otherwise it's in danger of being removed